keps/sig-windows/0000-20190103-windows-node-support.md

```diff
@@ -106,6 +107,196 @@ As of 29-11-2018 much of the work for enabling Windows nodes has already been co
 ## Implementation History
 
 
+## Testing Plan
+
+The testing for Windows nodes will include multiple approaches:
+
+1. [Adapting](#Adapting-existing-tests) some of the existing conformance tests to be able to pass on multiple node OS's
+2. Adding [substitute](#Substitute-test-cases) test cases where the first approach isn't feasible or would change the tests in a way is not approved by the owner. These will be tagged with `[SIG-Windows]`
+3. Last, gaps will be filled with [Windows specific tests](#Windows-specific-tests). These will also be tagged with `[SIG-Windows]`
+
```

Comment:
I think it is important to note that any test that is tagged with `[SIG-Windows]` should always pass on windows. we should specify this in the text.

In addition, is it possible to tag any of the tests that are not applicable to windows with any tag? that way it is easy to differentiate which of the existing conformance tests are not applicable to windows (beyond the ones that were substituted)